docs/docs/general/06-Tutorials/google-cat-search.md

```diff
@@ -117,7 +117,7 @@ describe('jest with askui', () => {
       .typeIn("cat")
       .textfield()
       .below()
-      .logo().withText("G00g.e")
+      .icon().withText("G00g.e")
```

Comment:
is this still true with the easy ocr

Reply:
There is no logo() anymore so icon() is the way to go, isn't it?

Reply:
sorry, my comment was not clear, i was speaking about the `withText("G00g.e")`.  is the easy ocr detecting it like that? 
and we don't have a such an icon called "G00g.e". this will cause this step to always fail. maybe use text instead of an icon

Reply:
I tested it and corrected the faulty instruction 😀